includes/admin/class-wc-payments-admin.php

```diff
@@ -277,9 +306,19 @@ public function enqueue_payments_scripts() {
 		}
 
 		// TODO: Update conditions when ToS script is enqueued.
-		if ( false && $current_tab && $current_section
-		&& 'checkout' === $current_tab
-		&& 'woocommerce_payments' === $current_section ) {
+		$show_on_payment_methods_list = (
+			$current_tab
+			&& 'checkout' === $current_tab
+			&& isset( $_GET['tos-disabled'] ) // phpcs:ignore WordPress.Security.NonceVerification
+		);
+
+		$show_on_settings_page = (
+			$current_tab && $current_section
+			&& 'checkout' === $current_tab
+			&& 'woocommerce_payments' === $current_section
+		);
+
+		if ( $show_on_payment_methods_list || $show_on_settings_page ) {
 			wp_enqueue_script( 'WCPAY_TOS' );
```

Comment:
Is it correct that this line will do nothing if script is not registered?

Reply:
Probably yes, but since https://github.com/Automattic/woocommerce-payments/pull/993/commits/8ad28b977f3e8bb150397473aadadaf845edfb29#diff-3863b4f88d0515e080fadc56af3c5e38e942afb96e569cb000ae6a594e8bea4aR289 the script is always registered.